develop/TOOLS/DIS/Troubleshooting/DIS_Troubleshooting_LicenseIssue.md

```diff
@@ -0,0 +1,30 @@
+---
+uid: DIS_Troubleshooting_LicenseIssue
+keywords: dis license, account not licensed
+---
+
+# Licensing problem
+
+## Problem
+
+Even though DIS is freely available, it can occur that the license check fails.
+
+## Possible solutions
+
+1. Check if you can reach [SkylineAPI](https://api.skyline.be/Help).
+
+   If not, check the certification and make sure it is allowed in your network.
+
+1. Check if you can you log in to SkylineAPI.
+
+   If not, check if you are using the right credentials.
+
+1. If the issue persists, check if your PC is still in sync with the time server.
+
+   Try disabling the synchronization and enabling it again, because the information shown by Windows is not always correct.
+
+If the issue still persists after you have tried the solutions above, report the issue to Skyline, making sure to include the following information:
```

Comment:
Which email address should they contact? It would be more helpful to immediately specify this here.

Reply:
See other remark